index.html

```diff
@@ -160,20 +160,41 @@ <h2>Page Layout</h2>
                     </select>
                 </span>
 
-                <span class="row"> <label for="center_margins">Margins</label>
-                    <select name="center_margins" id="center_margins" disabled>
-                        <option value="spine" selected>Preserve spine fold margin</option>
-                        <option value="center">Center</option>
-                    </select>
-                    <span data-balloon-length="medium" aria-label="If you have designed your input pdf with the intention of trimming printed pages down, you may wish to preserve the spine margin." data-balloon-pos="up">ℹ️</span>
+                <span class="row">
+                    <div style="width:225px;position: absolute;margin-left: 250px;font-size: smaller;">
```

Comment:
Not worth holding up this PR for it, but I think we should try to move these inline styles into the main CSS file at some point.

Reply:
no, you're absolutely right -- I went back and fixed most the inlined CSS for this section & will try to do better/fix the other `style`s when I encounter them in future PRs